javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @class listNoteView
+ * @description manage view of listNote
+ */
+export default class ListNoteView {
+  constructor(noteView) {
+    this.noteView = noteView;
+    this.listNoteElement = document.querySelector('.list-notes');
+
+    // input form
+    this.formElement = document.querySelector('.form-add-note');
```

Comment:
Duplicate `document.querySelector(` which cause long code. You could create an util function in folder `utils` or `helpers` , named may be `selectDOMByClass`  and use through all places.